module1.js

```diff
@@ -69,15 +75,20 @@ function question1(name1) {
   @returns [array 0 ... 100] 
 */
 
-// function question4() {
-//   // TODO
-//   for ( /* ??? */ ) {
-//     // TODO
-//   }
-//   return /* ??? */;
-// }
-
-
+function question4() {
+  // TODO
+  var num=[]
+          var arr =[]
+          for ( var i = 0; i<=100; i++ ) {
+              num.push(i);
+          }
+
+            // TODO
+    
+          arr = num.filter(x => x % 2 === 0);
+          return arr
+          }
+        
 /**
   +++++++++++++++++++++++++++++++++++++++++++
```

Comment:
You can shorten this solution up a bit, if you increment `i` by 2 instead of 1, like so...
```
for (var i = 0; i < 100; i+=2)
```
that way, you wouldn't need to use the filter statement, and you would also remove the need for the `arr` variable.

Reply:
okay i will do that